index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet">
+    <title>Cash Register</title>
+    <style>
+        body {
+          background: linear-gradient(to right, purple, orange, navy);
+          color: white;
+          font-family: Arial, sans-serif;
+          margin: 0;
+          padding: 0;
+        }
+
+        header {
+          text-align: center;
+          padding: 20px;
+        }
+
+        h1 {
+          font-size: 48px;
+          margin: 0;
+          color: navy;
+        }
+
+        table {
+          border-collapse: collapse;
+          width: 100%;
+          margin-bottom: 20px;
+        }
+
+        th, td {
+          padding: 10px;
+          text-align: left;
+        }
+
+        th {
+          background-color: purple;
+          color: white;
+        }
+
+        tr:nth-child(even) {
+          background-color: navy;
+        }
+
+        #icons {
+          display: flex;
+          justify-content: space-evenly;
+          margin-bottom: 20px;
+        }
+
+        #icons img {
+          width: 50px;
+          height: 50px;
+        }
+
+        #buttons {
+          display: flex;
+          justify-content: space-evenly;
+        }
+
+        button {
+          padding: 10px 20px;
+          background-color: navy;
+          color: white;
+          border: none;
+          cursor: pointer;
+        }
+
+        button:hover {
+          background-color: purple;
+        }
+    </style>
```

Comment:
- You have a different file for your CSS codes which is `style.css` and here also you linked a file named `styles.css` please change the name of your `style.css` to `styles.css` and move your CSS code there.